tests/test_pyrocky.py

```diff
@@ -34,8 +34,40 @@ def rocky_session():
     rocky.close()
 
 
-def test_minimal_simulation(rocky_session, tmp_path):
+@pytest.fixture()
+def rocky_cleanup(request):
+    def cleanup():
+        from ansys.rocky.core.client import _ROCKY_API
+
+        global _ROCKY_API
+
+        if _ROCKY_API is not None:
+            _ROCKY_API.Exit()
+
+    request.addfinalizer(cleanup)
+
+
+@pytest.mark.parametrize(
+    "version, expected_version",
+    [
+        ("25.1", 251),
+        ("24.2", 240),
+    ],
+)
+def test_minimal_simulation(version, expected_version, tmp_path, rocky_cleanup):
+    """Minimal test to be run with all the supported Rocky version to ensure
+    minimal backwards compatibility.
+    """
+    short_v = version.replace(".", "")
+    exe_file = f"C:\\Program Files\\ANSYS Inc\\v{short_v}\\Rocky\\bin\\Rocky.exe"
+    pyrocky.launch_rocky(exe_file)
     rocky = pyrocky.connect_to_rocky()
+
```

Comment:
I'd say if you add `request.addfinalizer(rocky.close)` here, we could discard the `rcoky_cleanup` fixture.